src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -117,8 +118,13 @@ static const std::unordered_map<std::type_index, cudnnReduceTensorOp_t> reduce_m
     {TI(ngraph::op::Add), CUDNN_REDUCE_TENSOR_ADD},
     {TI(ngraph::op::Multiply), CUDNN_REDUCE_TENSOR_MUL},
     {TI(ngraph::op::Maximum), CUDNN_REDUCE_TENSOR_MAX},
-    {TI(ngraph::op::Minimum), CUDNN_REDUCE_TENSOR_MIN},
-};
+    {TI(ngraph::op::Minimum), CUDNN_REDUCE_TENSOR_MIN}};
+
```

Comment:
Are these static unordered_maps used in multiple functions? 

If not, thinking we could declare them (statically) inline in the function call, rather than having them be at global scope.

Reply:
Good point, fixed.